src/gui/properties/propertieswidget.cpp

```diff
@@ -309,6 +309,10 @@ void PropertiesWidget::loadTorrentInfos(BitTorrent::Torrent *const torrent)
         m_ui->labelCommentVal->setText(Utils::Misc::parseHtmlLinks(m_torrent->comment().toHtmlEscaped()));
 
         m_ui->labelCreatedByVal->setText(m_torrent->creator());
+
+        // Private
+        m_ui->labelPrivateVal->setText((m_torrent && m_torrent->isPrivate()) ? tr("Yes") : tr("No"));
```

Comment:
Does it depend on metadata? If not, it should be moved out (above) of the if conditional.

Reply:
This check is redundant anyway:
```suggestion

        // Private
        m_ui->labelPrivateVal->setText(m_torrent->isPrivate() ? tr("Yes") : tr("No"));
```

Reply:
> Does it depend on metadata? If not, it should be moved out (above) of the if conditional.

AFAIK, we can't know whether torrent is private or not if there is no metadata available. So I would display "N/A" in case when there are no metadata available.